app/util.ts

```diff
@@ -1,11 +1,43 @@
 import { type } from "os";
 
 export const getLocationLink = (content: string) => {
-    const isMITLocationTest = new RegExp(".*-.*\\d$");
-    const isLocation = isMITLocationTest.test(content)
-    return isLocation ? `https://whereis.mit.edu/?go=${content}` : ""
-}
+  const isMITLocationTest = new RegExp(".*-.*\\d$");
+  const isLocation = isMITLocationTest.test(content);
+  return isLocation ? `https://whereis.mit.edu/?go=${content}` : "";
+};
 
 export const compareIgnoreCase = (str1: string, str2: string) => {
-    return str1.toUpperCase() == str2.toUpperCase();
-}
\ No newline at end of file
+  return str1.toUpperCase() == str2.toUpperCase();
+};
+
+function removeBase64(input: string) {
```

Comment:
Is this actually needed? Presumably the email has already been parsed and the email only contains the HTML.